public/kcl-samples/food-service-spatula/main.kcl

```diff
@@ -31,7 +31,7 @@ fn slot(sketch1, start, end, width) {
       toDegrees(      atan((end[1] - start[1]) / (end[0] - start[0])))
     }
   }
-  dist = sqrt(pow(end[1] - start[1], 2) + pow(end[0] - start[0], 2))
+  dist = sqrt(pow(end[1] - start[1], exp = 2) + pow(end[0] - start[0], exp = 2))
```

Comment:
Good candidate for a KCL function imo. `std::distance(a: point, b: point) -> length`